stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/RepositoryFacade.kt

```diff
@@ -26,7 +27,7 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.awaitAll
 import java.util.Date
 
-internal class RepositoryFacade constructor(
+internal class RepositoryFacade @VisibleForTesting internal constructor(
```

Comment:
Do we still need an internal constructor here?

Reply:
We use it in the `BaseRepositoryFacadeTest`